src/power_grid_model/validation/rules.py

```diff
@@ -70,12 +70,12 @@
     ValidationError,
 )
 from power_grid_model.validation.utils import (
-    eval_expression,
-    get_indexer,
-    get_mask,
-    get_valid_ids,
-    nan_type,
-    set_default_value,
+    _eval_expression,
+    _get_indexer,
```

Comment:
Isn't this one intended to be public?

Reply:
we want people to use `PowerGridModel.get_indexer()` instead (https://power-grid-model.readthedocs.io/en/stable/api_reference/python-api-reference.html#power_grid_model.PowerGridModel.get_indexer). 

In addition, the `power_grid_model.validation.utils` module where it is declared has a module-level docstring that says
```
Utilities used for validation. Only errors_to_string() is intended for end users.
```

This file actually exposes these utilities even though it only wishes to use it internally

Reply:
Oops I confused this one with the other `get_indexer`. Great find.

Reply:
@zhen0427 You can implement this in sensitivity directly instead of importing.

Reply:
@nitbharambe @zhen0427 but `power_grid_model.get_indexer` is still public. Or is it different to the other indexer in the validation functions?